engine/src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -247,6 +247,31 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures with
       } yield ()).futureValue
     }
 
+    it should "return pagination metadata only when page and pagesize query params are specified" taggedAs DbmsTest in {
+      assume(canConnect || testRequired)
+      val workflowInfo = materializeWorkflowDescriptorFromSources(workflowSources = testSources)
+      (for {
+        _ <- dataAccess.createWorkflow(workflowInfo, Nil, Nil, localBackend)
+        //get metadata when page and pagesize are specified
+        _ <- dataAccess.queryWorkflows(
+          WorkflowQueryParameters(Seq(WorkflowQueryKey.Page.name -> "1", WorkflowQueryKey.PageSize.name -> "50"))) map { response =>
```

Comment:
rather than using response._1 and response._2, can you match the tuple as `(response, meta)`?

Something like `map { case (response, meta) => ...}` So you can avoid all the tuple accesses